MIPS/mip-0005-mip-upgrade.md

```diff
@@ -0,0 +1,315 @@
+- Assess the risk of a MIP to the long-term health of the protocol.
+- Publish a Risk Assessment with a rationale.
+- Join the meetings scheduled by the MIP Facilitator.
+
+#### New Role Frameworks
+
+To ensure accountability and transparency for MIP Facilitators and Reviewers, we propose developing frameworks that clearly explain the processes for role selection, assignment, removal, resignation, and compensation. 
+
+These frameworks will share common principles, including:
+
+- Any community member can propose themselves as a candidate for MIP Facilitator or MIP Reviewer by submitting a new MIP.
+- Any community member can propose the removal of an existing MIP Facilitator or MIP Reviewer by submitting a new MIP.
+- Any community member can vote to approve or reject the selection or removal of MIP Facilitators and MIP Reviewers.
+- As the community builds its capacity to effectively take on these responsibilities, key ecosystem players will continue to fulfill these roles as needed, supporting the bootstrapping and testing of the new processes.
+
+#### Bootstrapping the Process
+
+However, due to the limited current capacity we propose that the Mina Foundation, in collaboration with o1 Labs and other members of the Mina ecosystem, will temporarily fill these new roles as needed while the processes are tested, and the capacity and expertise grows in the community.
+
+This bootstrapping period would last no longer than 12 months.
+
+### Introducing Mass Deliberation Tools and Public Debates
+
+We propose to implement **new tools** to collect and analyze the comments and sentiments from the community at different phases in the MIP process. These include:
+
+- Polling on MinaResearch. The community is encouraged to experiment with different methods to increase effectiveness.
+- Survey bot on Discord. The development and usage of open source AI bots on Discord to gather, organize and summarize mass feedback. The community is encouraged to review and propose improvements to the development:
+https://github.com/MinaFoundation/gptSurveySummarizer 
+- On-Chain Voting. The community is encouraged to suggest any improvements that could be proposed in a future MIP:
+https://github.com/MinaFoundation/mina-on-chain-voting/tree/main 
+https://github.com/MinaFoundation/govbot 
+- Governance Dashboards. The community is encouraged to experiment with different methods and tools to increase participation in the MIP process.
+https://minascan.io/mainnet/governance/MIPs 
+
+We propose to introduce new types of public debates:
+
+- During the expert part of the Review Phase: The MIP Facilitator will schedule, livestream and record open and public meetings between the MIP Author, the MIP Review Group and other stakeholders. The meeting minutes will be posted and shared in the designated channels for public consumption, allowing the community members who are interested to follow the process and contribute with comments. 
```

Comment:
Chatham house rules specified elsewhere may conflict with public deliberation.

Reply:
I agree that public deliberation is the default and preferred mechanism for fostering transparency and community trust. However, I acknowledge that certain discussions may require confidentiality under specific circumstances, such as the following examples that crossed my mind:

- Security concerns: Discussions around protocol vulnerabilities or other sensitive technical topics may need to be private to prevent exposing risks to potential attackers prematurely.
- Facilitation needs: Reviewers or stakeholders might find it beneficial to privately align on particularly sensitive or controversial topics before publishing their risk assessments or decisions.

In such cases, applying Chatham House Rules judiciously could provide a balance between transparency and the need for discretion. This could also ensure that when confidentiality is necessary, it is handled responsibly and documented appropriately. I would love to hear thoughts from the community on defining clear guidelines for when exceptions should apply to uphold accountability.

Reply:
Perhaps a private deliberation can be invoked by unanimous consent of the author, facilitator,
And reviewers? That could be too high of a threshold - alternatively how about a super majority (propose 67% threshold) vote to have a private session that is not shared with the community until a later (safe) point in time?

I think security should be the only exception. Private "alignment" sounds exactly like the kind of thing that is better done in public.